snackbars/src/main/java/com/decathlon/vitamin/compose/snackbars/VitaminSnackbarsColors.kt

```diff
@@ -0,0 +1,19 @@
+package com.decathlon.vitamin.compose.snackbars
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+data class VitaminSnackbarsColor(
```

Comment:
Add `@Immutable` annotation for Compose optimization

Reply:
Remove `Vitamin` keyword, add `s` at the end of the class name and remove `s` at `Snackbars`

=> `SnackbarColors`